packages/devtools_app/lib/src/ui/icons.dart

```diff
@@ -303,3 +305,9 @@ class ColorIconMaker {
     return iconCache.putIfAbsent(color, () => ColorIcon(color));
   }
 }
+
+class OcticonIcon extends DevToolsIcon {
```

Comment:
Are any instances of this class getting created?

Reply:
yes, in service_extensions.dart above